README.md

```diff
@@ -60,6 +61,29 @@ fuck go back
 
 they're the same picture
 ```
+4. Express your feelings during a math exam:
+```
+upgrade
+what can I say except A
+fuck go back
+```
+
+5. Print the alphabet with spaces in between and a new line at the end:
+```
+eax is great, but I want 65
+
+upgrade
+what can I say except al
+what can I say except \s
+upvote eax
+corporate needs you to find the difference between eax and 91
+fuck go back
+
+they're the same picture
+what can I say except \n
+```
+
+You can also check the [examples](examples/) directory to find more.
```

Comment:
The files in the example directory have the .masm extension. However, I decided to go with .memeasm since .masm is already taken by the [Microsoft Macro Assembler](https://docs.microsoft.com/en-us/cpp/assembler/masm/microsoft-macro-assembler-reference?view=msvc-160)

Reply:
https://github.com/kammt/MemeAssembly#compiling-and-running-your-code

> Keep in mind that it is recommended to save your code in a .memeasm file